addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -222,13 +222,18 @@ function clickOnEditAndWaitEditModeInTranslatedPage(position = "bottom") {
  */
 function clickOnSnippet(snippet, position = "bottom") {
     const trigger = snippet.id ? `#wrapwrap .${snippet.id}` : snippet;
-    return {
-        trigger: `:iframe ${trigger}`,
-        extra_trigger: "body.editor_has_snippets",
+    return [
+        {
+            trigger: "body.editor_has_snippets",
+            noPrepend: true,
+        },
```

Comment:
- What is the addition of `noPrepend` about?
- If you don't define a `run` it now means it does nothing? :thinking: Are not you supposed to add `isCheck: true` for that? :thinking:

Reply:
If you don't define a run it now means it does nothing? Yes ! 
Are not you supposed to add isCheck: true for that? 🤔 
isCheck no longer exists. The default action is "Nothing". (So just checking if element is in DOM)

Reply:
What is the addition of noPrepend about?
Because a function in website/.../tour_utils.js that use prepend ... only used in website. 
```
function prepend_trigger(steps, prepend_text='') {
    for (const step of steps) {
        if (!step.noPrepend && prepend_text) {
            step.trigger = prepend_text + step.trigger;
        }
    }
    return steps;
}
```

Reply:
- And it checks for its visibility in all cases then?
- What about the `auto` option, does it still exist? I guess it is automatically `true` when there is no `run` then?

Reply:
Let's see https://github.com/odoo/odoo/pull/168829

Reply:
There is no change at the moment regarding the trigger visibility check. However, in the coming weeks, we will completely remove the visibility check to find the trigger in the DOM. However, to perform an action (such as editing a field) we will then check whether the trigger is visible or not.
The same will apply to disabled elements.

Reply:
If you want to check an element is visible in DOM ... ( or enabled ) you'll need to use pseudo-selector
:enable
:visible

Reply:
> Not implicit ..
> [...]
> If tour is in manual mode and that the step has no run and has no consumeEvent => the step is skipped

So... implicit `isActive: ["auto"]` no? :flushed: 

= We have to be sure that the same tour using those website utils can be used for users and for automatic tests, but the extra_trigger step must only be considered during automatic tests. So an implicit `isActive: ["auto"]`.